src/OpenSage.Game/Utilities/LanguageUtility.cs

```diff
@@ -39,6 +39,9 @@ public static string ReadCurrentLanguage(IGameDefinition gameDefinition, FileSys
                 case SageGame.Bfme2:
                 case SageGame.Bfme2Rotwk:
                     return DetectFromFileSystem(fileSystem, "lang", "", "Audio.big");
+                case SageGame.Ra3Uprising:
+                case SageGame.Ra3:
+                    return DetectFromFileSystem(fileSystem, "Data", "", "Audio.big");
```

Comment:
Some languages doesn't have their own localized audio files, IIRC Traditional Chinese has `ChineseT.big` in RA3, but it still uses `EnglishAudio.big` for audio data